Zend/zend_highlight.c

```diff
@@ -134,11 +127,11 @@ ZEND_API void zend_highlight(zend_syntax_highlighter_ini *syntax_highlighter_ini
 
 		if (last_color != next_color) {
 			if (last_color != syntax_highlighter_ini->highlight_html) {
-				zend_printf("</span>");
+				zend_printf("</code>");
```

Comment:
Those should still be ``span`` elements

Reply:
I guess you are right. I checked how it works on Stack Overflow and they do it exactly like this. I changed it now.